src/components/tipRecords/TipRecord.vue

```diff
@@ -34,8 +34,8 @@
         class="tip__article"
       >
         <a
-          :href="tip.url"
           target="_blank"
+          @click="goToUrl(tip.url)"
```

Comment:
why is that needed?
This way you can't see the link tooltip of the browser, also the `<a>` tag becomes no-different than a `div`

Reply:
Because we prevent the click action from bubbling, otherwise the link gets its href executed and always open a new tab. In this case we want to open the dialog

Reply:
It should not happen if you use `@click.stop="handler"`